jobs/JRTMA3D_UPDATEVARS

```diff
@@ -131,7 +131,7 @@ if [ "${envir}" != "esrl" ]; then #WCOSS
     fi
     export DATAHOME=${DATA_RUNDIR}/gsiprd
 #    ${RM} -rf ${DATAHOME}
-    ${LN} -s  ${DATA}  ${DATAHOME}
+#    ${LN} -s  ${DATA}  ${DATAHOME}
```

Comment:
@MatthewMorris-NOAA @AnnetteGibbs-NOAA @EdwardColon-NOAA 
Hi all,

I am not sure if this line is commented off, will this script still be working? Or actually this line is not necessary?

-Gang

Reply:
@MatthewMorris-NOAA 
if you have tested it and there is no problem in the parallel test, then it is good to go.

Reply:
@GangZhao-NOAA One of these variables is not defined, so this line currently creates a symbolic link to the gsiprd in the user's home directory.

Reply:
Great! Thank you for the explanation.